### PR TITLE
Assignment 4

### PR DESCRIPTION
**Pull request by t-kanin**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "editor.tabSize": 2,
+    "editor.detectIndentation": false
+}
```
Comment (dolwijit13) [inline .vscode/settings.json]: You can add `"files.insertFinalNewline": true,` to auto insert a newline at end of file.
And you can add this in **cmd + shift + p -> open settings**

File: assignment_4.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+def solution(day, k)
+  days = %w[Mon Tue Wed Thu Fri Sat Sun]
+  index = days.index(day)
+  index = (index + k) % 7 
+  pp index
```
Comment (dolwijit13) [inline assignment_4.rb]: Remove this?

File: assignment_4.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+def solution(day, k)
+  days = %w[Mon Tue Wed Thu Fri Sat Sun]
```
Comment (WatcharitNew) [inline assignment_4.rb]: Maybe move to outside function like this
```suggestion
DAYS = %w[Mon Tue Wed Thu Fri Sat Sun]

def solution(day, k)
```

File: assignment_4.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+DAYS = %w[Mon Tue Wed Thu Fri Sat Sun]
```
Comment (aunsira) [inline assignment_4.rb]: We normally put `freeze` after value of constant to avoid mutation. https://ruby-doc.org/core-2.7.3/Object.html#freeze-method

**Review by aunsira — APPROVED** [review]

**Pull request merged**